addons/mail/static/src/core/common/external_window_service.js

```diff
@@ -0,0 +1,86 @@
+import { registry } from "@web/core/registry";
+import { reactive } from "@odoo/owl";
+
+
+export const externalWindowService = {
+    dependencies: ["ui"],
+    start(env, { ui }) {
+        const anchor = document.createElement("div");
```

Comment:
I would rename service to `mail.popout`: shorter and follows convention of service having module name as prefix.